_aggregations/pipeline/percentiles-bucket.md

```diff
@@ -0,0 +1,282 @@
+---
+layout: default
+title: Percentiles bucket
+parent: Pipeline aggregations
+nav_order: 160
+redirect_from:
```

Comment:
You don't need a redirect unless there was a file already with this topic in a different location. In this case, since this is new, no redirect needed.

Reply:
Makes sense. Going to have to remove a few of these then, I reckon. 😁